scripts/e

```diff
@@ -300,7 +300,14 @@ case "$1" in
     mkdir "$OUT_DIR/libreofficekit"
 
     echo "Bringing over local libreofficekit"
-    cp -rf "$LOK_PATH/libreoffice-core/instdir/*" "$OUT_DIR/libreofficekit/"
+
+    if [ "$OS" = "mac" ]; then
+      if [[ $(uname -m) == 'arm64' ]]; then
```

Comment:
I recommend installing [shellcheck](https://www.shellcheck.net/), avoids weird little bugs like string escaping.

```suggestion
    if [ "$OS" = "mac" ] && [ "$(uname -m)" = "arm64" ]; then
```

You can do and/or using `&&` and `||` between tests. String comparisons can be done using `[ "x" = "x" ]`, that is a single `=` and a single pair of brackets.

I noticed this doesn't handle the case for x64, otherwise keeping it as two separate if statements is fine, since there is an `else` case.

---

That being said, for development purposes locally, I've mostly stuck to using `ln -s`. I'll make a PR for the updates to the README tomorrow.

Reply:
RE: the README update, I forgot I already updated it :joy: 

https://github.com/coparse-inc/electron-libreoffice#how-do-i-update-libreofficekit-after-compiling-it-locally